src/main/java/com/northconcepts/datapipeline/examples/cookbook/SerializeAndDeserializeRecords.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2006-2020 North Concepts Inc.  All rights reserved.
+ * Proprietary and Confidential.  Use is subject to license terms.
+ * 
+ * https://northconcepts.com/data-pipeline/licensing/
+ */
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.math.BigInteger;
+
+import com.northconcepts.datapipeline.core.Record;
+
+public class SerializeAndDeserializeRecords {
+
+	public static void main(String[] args) throws Throwable {
+		Record record1 = new Record();
```

Comment:
More indentation issues.

Reply:
converted indentation to spaces and fixed it.